### ISSUE TITLE
question(YouTube - Change shorts repeat state): What is the difference between `Repeat` and `Default`?

### ISSUE THREAD

**Issue by odkate** [inotia00/ReVanced_Extended#2083]
### Application

YouTube

### Question description

I don't see any difference between these repetition states of Shorts

### Acknowledgements

- [X] This issue is not a duplicate of an existing question.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by inotia00** [comment]
`Default` is literally equivalent to `off`.

For example:
- Autoplay is used for general shorts.
- In shorts livestreams, the video pauses when the livestream ends.
- Some shorts are played only once (Some Shorts ads or Shorts survey)

If you change the `Change shorts repeat state` setting to `Autoplay`, autoplay will always be applied to shorts.

For example, even if the livestream ends, the video does not pause and automatically plays to the next livestream.
